pom.xml

```diff
@@ -90,9 +90,11 @@
     <slf4j.version>1.7.36</slf4j.version>
     <taglibs-standard-impl.version>1.2.5</taglibs-standard-impl.version>
     <taglibs-standard-spec.version>1.2.5</taglibs-standard-spec.version>
-    <testcontainers.version>1.19.6</testcontainers.version>
+    <testcontainers.version>1.19.7</testcontainers.version>
     <weld.version>3.1.9.Final</weld.version>
-    <wildfly.elytron.version>2.3.1.Final</wildfly.elytron.version>
+    <!-- Stay on 2.2.x as 2.3.x is for Java 11 -->
+    <wildfly.elytron.version>2.2.4.Final</wildfly.elytron.version>
```

Comment:
shouldn't we have an entry for this restriction in `dependabot.yml` file?

Reply:
Probably, but this isn't the only one.